dev/release/verify-release-candidate.sh

```diff
@@ -414,12 +416,12 @@ test_glib() {
 test_js() {
   pushd js
 
-  export NVM_DIR="`pwd`/.nvm"
-  mkdir -p $NVM_DIR
-  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
-  [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-
-  nvm install node
+  # export NVM_DIR="`pwd`/.nvm"
+  # mkdir -p $NVM_DIR
+  # curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+  # [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+  #
+  # nvm install node
```

Comment:
This causes problems?

Reply:
Something something npm version conflict. GitHub Actions runners already have new enough versions of node/npm installed so we don't need to install it in the verification script. FWIW none of the other languages install themselves in the script, they describe their requirements as prerequisites on the wiki.

Reply:
Hmm, annoying. I made Go install itself recently (now I notice that the docs at the top of the file are out of date). I had added the Node bootstrap on account of Node versions in OS package managers not being new enough generally

Reply:
The wiki has these nvm instructions on it, so this is redundant to that at least. 

I'd be in favor of having a separate setup script for installing necessary dependencies for running verification tests (or really, for getting a working setup for all Arrow subprojects), perhaps wired into some docker(-compose) setup.